jsdoc.json

```diff
@@ -10,8 +10,8 @@
             "./src",
             "README.md",
             "src/react-guides",
+            "src/react-ruler",
             "node_modules/@scena/event-emitter/src",
-            "node_modules/@scena/react-ruler/src/react-ruler"
```

Comment:
we probably don't need to have this file. it's for the documentation but we don't support this

Reply:
I think the file can be removed in the scope of this ticket

Reply:
or we have to think about the documentation to leave this.
because when I was refactoring guides I removed all those comments

Reply:
Jsdoc removed